src/components/calcite-popover/resources.ts

```diff
@@ -0,0 +1,4 @@
+export const CSS = {
```

Comment:
Maybe unneeded abstraction now? Unless we want to do this for our other components as well 🤷‍♂

Reply:
The reason we did this was to share the CSS classes for use within tests. Since Stencil only lets you export one thing we had to have a seperate file. Let me know if you don't think its worth it. It just keeps the classes in sync for tests.

Reply:
👍 to keeping the tests as robust as possible. Since the tests are using classnames for selecting elements, having those defined in a common shared place is helpful to keep classname renames from breaking tests. An alternative could be to use data attributes for test selectors, but so long as we're using classnames then this pattern ensures they remain consistent.